server/utils.go

```diff
@@ -104,6 +104,9 @@ func newPodNetwork(sb *sandbox.Sandbox) ocicni.PodNetwork {
 		Namespace: sb.Namespace(),
 		ID:        sb.ID(),
 		NetNS:     sb.NetNsPath(),
+		//PortMappings: sb.PortMappings(),
```

Comment:
Is this commented out because we don't have support for it yet?

Reply:
Yes along with us not using ocicni for port bindings but rather a k8s library. Though I think I meant to remove that comment